src/server/rest/v1/router/api/BillingRouter.ts

```diff
@@ -96,6 +97,13 @@ export default class BillingRouter {
     });
   }
 
+  private buildRouteForceSynchronizeUser(): void {
+    this.router.patch(`/${RESTServerRoute.REST_BILLING_USER_SYNCHRONIZE}`, (req: Request, res: Response, next: NextFunction) => {
+      req.body.id = req.params.id;
```

Comment:
@ClaudeROSSI let me know if you want to use the directly the req.params in the filteredrequest.
Meanwhile it works this way.